PhysicsTools/PatAlgos/python/cleaningLayer1/tauCleaner_cfi.py

```diff
@@ -8,7 +8,7 @@
         'tauID("decayModeFinding") > 0.5 &'
         ' tauID("byLooseCombinedIsolationDeltaBetaCorr3Hits") > 0.5 &'
         ' tauID("againstMuonTight3") > 0.5 &'
-        ' tauID("againstElectronVLooseMVA5") > 0.5'
+        ' tauID("againstElectronVLooseMVA6") > 0.5'
```

Comment:
Hi @slava77 
yes this change makes sense and should be applied. MVA6 replaced MVA5 with same functionality and purpose.
Cheers,
Roger